scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>I think this cell needs to be above the import statements, similar to e.g. object detection. We noticed some problems otherwise.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>